articles/discord-bot-node/index.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/discord-bot-node/
+title: So You Want to Make a Discord Bot (With Node.js)
+description: Discord bots can do lots of things. They can manage voice chats, moderate servers, and make lots of fun. Making them is also fun. This tutorial shows you how to make your own.
+author: mike-white
+date: 2020-08-13T00:00:00-07:00
+topics: [Node.js]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/discord-bot-node/hero.jpg
+    alt: discord bot node example image
+---
+Discord bots are very cool. They're also very fun to make. But how do you make one?
+
+Typically, Discord bots are written in [Node.js](https://nodejs.org/en/). Node is a JavaScript interpreter that runs on the desktop. Of course, it's possible to write a bot in [other languages](https://discordapi.com/unofficial/libs.html). For some people this may be preferable. If you have lots of users, you will need an efficient language ([unlike javascript](https://benchmarksgame-team.pages.debian.net/benchmarksgame/fastest/node-gpp.html)). We will use the JavaScript library because it is the most popular. But, many of the other libraries are somewhat similar. Once you are done reading this article, try looking at some [Rust tutorials](https://www.youtube.com/watch?v=sOA6rSRCqPw&list=PLPwSz_Jcam3xVjrTAYgIHvf1Jq94yrRXp).
+<!--more-->
+## Get Started By Making a Bot
+
+### Creating The Bot
+First, you need to visit the [Discord Developer Portal](https://discordapp.com/developers/applications/). Click the "New Application" button. Give your bot a name, and click "Create".
+
+![The New Application button is in the top right corner](./discordbot1.png)
+
+![Create](/engineering-education/discord-bot-node/discordbot2.png)
+
+You'll get to your application's page, where if you want, you can give it a description and an avatar.
+
+On the left, you should see a "Bot" tab. Click on that, and then click on the button that says, "Add Bot".
```

Comment:
I would separate out these directions so that the directions are in line with the images - rewrite the "then click on the button that says, "Add Bot"." bit and put it after your first image below.